packages/manager/src/utilities/analytics.ts

```diff
@@ -13,12 +24,12 @@ export const sendEvent = (eventPayload: AnalyticsEvent): void => {
   }
 
   // Send a Direct Call Rule if our environment is configured with an Adobe Launch script
-  if ((window as any)._satellite) {
+  if ((window as Window)._satellite) {
```

Comment:
Because we augmented the global window, I don't think we need the `as` anymore 🎉

```suggestion
  if (window._satellite) {
```

Reply:
ah yeah of course. done